src/main/java/com/danielagapov/spawn/Controllers/EventController.java

```diff
@@ -70,21 +70,23 @@ public ResponseEntity<List<? extends IEventDTO>> getEventsCreatedByUserId(@PathV
         }
     }
 
-    // full path: /api/v1/events/friendTag/{tagId}?full=full&requestingUserId=requestingUserId
+    // TODO: path is wrong
+    // full path: /api/v1/events/friendTag/{tagId}
     @GetMapping("friendTag/{tagId}")
-    public ResponseEntity<List<? extends IEventDTO>> getEventsByFriendTagId(@PathVariable UUID tagId, @RequestParam(value="full", required=false) boolean full, @RequestParam UUID requestingUserId) {
+    public ResponseEntity<List<? extends IEventDTO>> getEventsByFriendTag(@PathVariable UUID requestingUserId, @RequestParam(required=false) UUID tagId) {
```

Comment:
```suggestion
    public ResponseEntity<List<FullFeedEventDTO>> getEventsByFriendTagId(@PathVariable UUID friendTagId) {
```
1. the `? extends IEventDTO` is used if this endpoint needs to return either `EventDTO` or `FullFeedEventDTO`, but this one only needs to return the latter, so we can do that explicitly.
2. Per Slack, user id shouldn't be necessary, since in your new `getFilteredFeedEventsByFriendTagId()` should be able to gather that from the friend tag
3. Let's keep things simple from null checks by requiring `tagId`